common/pipeline_trigger/README.md

```diff
@@ -0,0 +1,153 @@
+<!-- Overview -->
+# Overview
+
+When using Azure Machine Learning(AML) Service, there are cases where it is necessary to operate the pipeline under various business scenario. This sample demonstrates a scenario where users without access to AML can trigger an AML pipeline using an azure function.
+
+**What does this sample demonstrate:**
+- How to invoke AML pipeline using Azure Function
+
+**What doesn't this sample demonstrate**:
+- Azure Machine Learning Service Creation.
+
+![architecture](media/architecture.png)
+
+<!-- REPO STRUCTURE -->
+# Repo Structure
+
+This repo contains sample code to trigger Azure ML pipelines from Azure Functions.
+
+The folders are structured as following -
+```
+common
+  └── pipeline_trigger
+        ├── devops_pipelines # Azure DevOps CI/CD pipeline definition
+        ├── environment_setup # Azure DevOps IaC pipeline definition to provision Azure resources
+        ├── media # images for README
+        │
+        ├── src
+        │    ├── PipelineHttpTrigger # scripts for pipeline trigger
+        │    ├── .funcignore
+        │    ├── host.json
+        │    ├── local.settings.json.example
+        │    └── requirements.txt
+        │
+        ├── tests
+        │      └── src
+        │           └── PipelineHttpTrigger
+        │                    └── test_pipeline_http_trigger.py # unit test for pipeline trigger
+        │
+        ├── .flake8 # configuration file for linter flake8
+        ├── .gitattributes
+        ├── .gitignore
+        └── README.md # explains what the sample is demonstrating and how to run it
+```
+
+<!-- ABOUT THE FOLDER -->
+## About The Folder
+
+This 'pipeline_trigger' folder contains source code for an Azure Function below:
+
+|Function|Purpose|
+|-|-|
+|PipelineHttpTrigger|HTTP trigger - This function will be triggered to run the AML pipeline when manually HTTP request including key is sent.|
+
+<!-- GETTING STARTED -->
+# Getting Started
+
+- [1. Create Azure Resources](##1.-Create-Azure-Resources)
+- [2. Set AML access right to Azure Function](##2.-Set-AML-access-right-to-Azure-Function)
+- [3. Option 1: Deploy the function project to Azure - Local](##3.-Option-1:-Deploy-the-function-project-to-Azure---Local)
+- [3. Option 2: Deploy the function project to Azure - CI/CD in Azure DevOps](##3.-Option-2:-Deploy-the-function-project-to-Azure---CI/CD-in-Azure-DevOps)
+- [4. How to use pipeline trigger](##4.-How-to-use-pipeline-trigger)
+- [How to renew a key](##How-to-renew-a-key)
+- [Linting and Testing](##Linting-and-Testing)
+
+## 1. Create Azure Resources
+Please refer to this [README.md](environment_setup\provisioning\README.md) under environment_setup/provisioning
+
+
+## 2. Set AML access right to Azure Function
+
+1. Enable System Assigned Identity
+  ![system_assigned_identity.png](media/system_assigned_identity.png)
```

Comment:
same here! I think all images are currently broken!

Reply:
Thanks @angie4u and @h2floh for reviewing this PR. 
Could you have a look at README on the branch? It seems not broken.
https://github.com/liupeirong/MLOpsManufacturing/tree/aykhara/pipelinetrigger/common/pipeline_trigger

Reply:
I can confirm that it's not broken on the branch

Reply:
I confirmed it too.